fvdb/viz/_camera_view.py

```diff
@@ -84,26 +87,61 @@ def axis_thickness(self, thickness: float):
         """
         self._view.axis_thickness = thickness
 
-    @property
-    def axis_scale(self) -> float:
-        return self._view.axis_scale
-
-    @axis_scale.setter
-    def axis_scale(self, scale: float):
-        self._view.axis_scale = scale
-
     @property
     def frustum_line_width(self) -> float:
+        """
+        Get the line width of the frustum in the camera frustum view.
+        """
         return self._view.frustum_line_width
 
     @frustum_line_width.setter
     def frustum_line_width(self, width: float):
+        """
+        Set the line width of the frustum in the camera frustum view.
+
+        Args:
+            width (float): The line width of the frustum in world units.
+        """
         self._view.frustum_line_width = width
 
     @property
     def frustum_scale(self) -> float:
+        """
+        Get the scale factor applied to the frustum visualization.
+        """
         return self._view.frustum_scale
 
     @frustum_scale.setter
     def frustum_scale(self, scale: float):
```

Comment:
The more general way (that we totally don't always follow) is to use `NumericScalarNative` instead of `float` here, which won't complain if you pass in e.g. np.float32